ofl/joan/DESCRIPTION.en_us.html

```diff
@@ -0,0 +1,8 @@
+<p>Inspired by the roman cut by Francesco Griffo, it is a tribute to the Italian style, a model to follow even in the 15th century.  
+Joan is characterized by neat serifs as well as sharp terminals and is intended for books and magazines.  
+The font supports the complete Latin Plus character set.  
```

Comment:
Again "Latin Plus" doesn't mean anything to user, better to speak in term of regions or not saying anything at all since the API will provide this information in the future.